github/util_v4_branch_protection.go

```diff
@@ -223,60 +232,89 @@ func branchProtectionResourceData(d *schema.ResourceData, meta interface{}) (Bra
 	return data, nil
 }
 
-func setDismissalActorIDs(actors []DismissalActorTypes) []string {
-	pushActors := make([]string, 0, len(actors))
+func setDismissalActorIDs(actors []DismissalActorTypes, d *schema.ResourceData, meta interface{}) []string {
+	dismissalActors := make([]string, 0, len(actors))
+	data, _ := branchProtectionResourceData(d, meta)
+	orgName := meta.(*Owner).name
+l:
 	for _, a := range actors {
-		if a.Actor.Team != (Actor{}) {
-			pushActors = append(pushActors, a.Actor.Team.ID.(string))
+		for _, v := range data.ReviewDismissalActorIDs {
+			if (a.Actor.Team.ID != nil && a.Actor.Team.ID.(string) == v) || (a.Actor.User.ID != nil && a.Actor.User.ID.(string) == v) {
+				dismissalActors = append(dismissalActors, v)
+				continue l
+			}
+		}
+		if a.Actor.Team.Slug != "" {
+			dismissalActors = append(dismissalActors, orgName+"/"+string(a.Actor.Team.Slug))
+			continue
 		}
-		if a.Actor.User != (Actor{}) {
-			pushActors = append(pushActors, a.Actor.User.ID.(string))
+		if a.Actor.User.Login != "" {
+			dismissalActors = append(dismissalActors, "/"+string(a.Actor.User.Login))
 		}
 	}
-
-	return pushActors
+	return dismissalActors
 }
 
-func setBypassPullRequestActorIDs(actors []BypassPullRequestActorTypes) []string {
+func setBypassPullRequestActorIDs(actors []BypassPullRequestActorTypes, d *schema.ResourceData, meta interface{}) []string {
 	bypassActors := make([]string, 0, len(actors))
+	data, _ := branchProtectionResourceData(d, meta)
+	orgName := meta.(*Owner).name
+l:
 	for _, a := range actors {
-		if a.Actor.Team != (Actor{}) {
-			bypassActors = append(bypassActors, a.Actor.Team.ID.(string))
+		for _, v := range data.BypassPullRequestActorIDs {
+			if (a.Actor.Team.ID != nil && a.Actor.Team.ID.(string) == v) || (a.Actor.User.ID != nil && a.Actor.User.ID.(string) == v) {
+				bypassActors = append(bypassActors, v)
+				continue l
+			}
 		}
-		if a.Actor.User != (Actor{}) {
-			bypassActors = append(bypassActors, a.Actor.User.ID.(string))
+		if a.Actor.Team.Slug != "" {
```

Comment:
Is this a breaking change for folks who weren't using the newly-introduced `/` character functionality?

Reply:
Got it, thanks!